cli/src/rdwatch_cli/__init__.py

```diff
@@ -172,8 +189,6 @@ async def movie(
             ]
             imgs[0].save(output, save_all=True, append_images=imgs)
         except ImageNotFound:
-            raise click.ClickException(
-                'No image found for the given bounding box and time'
-            )
+            print('No image found for the given bounding box and time')
```

Comment:
Why do we no longer need an exception here?

Reply:
Without it, the tile version will crash when creating a movie if a single image for a specific x/y/z location isn't found.  I'd much rather have it log and skip the missing the image (maybe I add in the timestamp/bbox) then not create a movie.

Reply:
Also, I'm not a fan of the current vocab of the endpoints.  'visual' means worldview is quite a leap, but I didn't want to alter the existing stuff so I used the same pattern.

Reply:
Ah I see, yeah I agree that it's better to skip the missing image instead of crashing. In that case, can we use `click.echo(...)` here instead of print?